    Object foo(Object o) throws MyException; // Compliant

